### PR TITLE
[admin-pod][image-fetcher][scorecard] use preemptible nodes

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

File: image-fetcher/deployment.yaml [hunk 1]
```diff
@@ -18,6 +18,8 @@ spec:
 {% if deploy %}
       priorityClassName: production
 {% endif %}
+      nodeSelector:
+        preemptible: "true"
```
Comment (cseed) [inline image-fetcher/deployment.yaml:22]: This is a daemon, it should be running on all nodes.  It should have a preemptible toleration, but no no selector.

File: scorecard/deployment.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ spec:
       priorityClassName: production
 {% endif %}
       nodeSelector:
-        preemptible: "false"
+        preemptible: "true"
```
Comment (cseed) [inline scorecard/deployment.yaml:23]: This needs a preemptible node toleraiton.

File: admin-pod/admin-pod.yaml [hunk 3]
```diff
@@ -12,6 +12,8 @@ spec:
       labels:
         app: admin-pod
     spec:
+      nodeSelector:
+        preemptible: "true"
```
Comment (cseed) [inline admin-pod/admin-pod.yaml:16]: This needs a preemptible node toleration.

**Comment by danking** [comment]
Done.

**Pull request merged**